stdlib/2and3/mailbox.pyi

```diff
@@ -0,0 +1,194 @@
+
+from typing import Optional, Union, Text, AnyStr, Callable, IO, Any, Iterator, List, Tuple, TypeVar, Protocol, Dict, Iterable, Generic, Type
+from types import TracebackType
+import sys
+import email
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+if sys.version_info >= (3, 6):
+    from builtins import _PathLike
+    _PathType = Union[bytes, Text, _PathLike]
+else:
+    _PathType = Union[bytes, Text]
+
+_T = TypeVar("_T")
+_MessageType = TypeVar("_MessageType", bound=Message)
+_MessageData = Union[email.message.Message, bytes, str, IO[AnyStr]]
```

Comment:
I don't think AnyStr makes much sense here because you're not using it as a TypeVar. Does just putting `IO[str]` as well as `IO[bytes]` in the Union fix the pytype problem?

Reply:
It does, according to my local test. I'll swap it over, as in this case it isn't using the TypeVar. I'll leave the pytype issue open though, as mypy accepts it for TypeVar cases (EG `def func(param: _Data[AnyStr]) -> AnyStr: ...`)